docs/release/condor-itb-testing.md

```diff
@@ -0,0 +1,142 @@
+# Testing HTCondor Prereleases on the Madison ITB Site
+
+This document contains a basic recipe for testing an HTCondor prerelease build on the Madison ITB site.
+
+## Prerequisites
+
+The following items are known prerequisites to using this recipe.  If you are not running the Ansible commands from
+osghost, there are almost certainly other prerequisites that are not listed below.  And even using osghost for Ansible
+and itb-submit for the submissions, there may be other prerequisites missing.  Please improve this document by adding
+other prerequisites as they are identified!
+
+* A checkout of the osgitb directory from our local git instance (not GitHub)
+* Your X.509 DN in the `osgitb/unmanaged/htcondor/condor_mapfile` file and (via Ansible) on `itb-ce1` and `itb-ce2`
+
+## Gather Information
+
+Technically skippable, this section is about checking on the state of the ITB machines before making changes.  The plan
+is to keep the ITB machines generally up-to-date independently, so those steps are not listed here.  And honestly, the
+steps below are just some ideas; do whatever makes sense for the given update.
+
+The commands can be run as-is from within the `osgitb` directory from git.
+
+1. Check OS versions for all current ITB hosts:
+
+        :::console
+        ansible current -i inventory -f 20 -o -m command -a 'cat /etc/redhat-release'
+
+2. Check HTCondor versions for all HTCondor hosts:
+
+        :::console
+        ansible condor -i inventory -f 20 -o -m command -a 'rpm -q condor'
+
+3. Obtain the NVR of the HTCondor prerelease build from OSG to test.  Do this by talking to Tim&nbsp;T. and checking
+   Koji.  The expectation is that the HTCondor prerelease build will be in the development repository (or
+   upcoming-development).
+
+## Install HTCondor Prerelease
+
+1. Shut down HTCondor and HTCondor-CE on prerelease machines:
+
+        :::console
+        ansible condordev -i inventory -bK -f 20 -m service -a 'name=condor-ce state=stopped' -l 'itb-ce*'
+        ansible condordev -i inventory -bK -f 20 -m service -a 'name=condor state=stopped'
+
+2. Install new version of HTCondor on prerelease machines:
+
+        :::console
+        ansible condordev -i inventory -bK -f 10 -m command -a 'yum --enablerepo=osg-upcoming --assumeyes update condor'
+
+    or, if you need to install an NVR that is “earlier” (in the RPM sense) than what is currently installed:
+
+        :::console
+        ansible condordev -i inventory -bK -f 10 -m command -a 'yum --enablerepo=osg-development --assumeyes downgrade condor condor-classads condor-python condor-procd blahp'
```

Comment:
We should think about specifying the condor version here as well

Reply:
Well, if necessary. My goal was not to create examples of every variation on doing the HTCondor installs. I figure that you all use yum enough to sort it out.